features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Configuration/components/APISecurity/APISecurity.jsx

```diff
@@ -92,7 +92,7 @@ export default function APISecurity(props) {
                 <Typography className={classes.bottomSpace}>
                     <FormattedMessage
                         id='Apis.Details.Configuration.components.APISecurity.allOptional'
-                        defaultMessage='Please select at least one API security level mandatory!'
+                        defaultMessage='Mandatory: Please select at least one API security level!'
```

Comment:
This should be fixed like this. https://github.com/wso2/carbon-apimgt/pull/7726/files